packages/edit-site/src/components/editor/index.js

```diff
@@ -196,6 +197,7 @@ function Editor() {
 
 	return (
 		<>
+			<PostRouter />
```

Comment:
I'm not sure if this naming is very helpful. as in my mind this is something like FSERouter or SiteEditorRouter... It can be addressed later though but wanted to get some thoughts from others..

Reply:
Yeah, I don't have any strong reasons for choosing `PostRouter` other than lack of creativity.  `EditorRouter` or something along the flavors you mentioned may be more fitting.

Reply:
Or would `QueryController`, or `URLQueryController` make more sense?  Any thoughts?